src/components/RadioButton/Bounceable.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react'
+import {ViewStyle, StyleProp, Pressable, PressableProps, View} from 'react-native'
+import Animated, {withSpring, useSharedValue, useAnimatedStyle} from 'react-native-reanimated'
+
+const AnimatedPressable = Animated.createAnimatedComponent(Pressable)
+
+type CustomStyleProp = StyleProp<ViewStyle> | Array<StyleProp<ViewStyle>>
+
+export interface IBounceableProps extends PressableProps {
+  /**
+   * onPress event
+   */
+  onPress?: () => void
+  /**
+   * The target value when onPressIn
+   * default: 0.93
+   */
+  bounceEffectIn?: number
+  /**
+   * The target value when onPressOut
+   * default: 1
+   */
+  bounceEffectOut?: number
+  /**
+   * The initial velocity of the animation in units per second when onPressIn
+   * default: 0.1
+   */
+  bounceVelocityIn?: number
+  /**
+   * The initial velocity of the animation in units per second when onPressOut
+   * default: 0.4
+   */
+  bounceVelocityOut?: number
+  /**
+   * The bounciness parameter determines how bouncy the animation should be when onPress
```

Comment:
I think you don't need to declare this prop because the PressableProps contains it